lib/Raven/Util/PHPVersion.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Raven\Util;
+
+class PHPVersion
+{
+    const VERSION_PARSING_REGEX = '/^(?<base>\d\.\d\.\d{1,2})(?<extra>-(beta|rc)-?(\d+)?(-dev)?)?/i';
+
+    /**
+     * @param string $version
+     * @return string
+     */
+    public static function getParsed($version = PHP_VERSION)
```

Comment:
I would rename this method to `parseVersion` so that it's more explicit. Also, we should add documentation about what the method and the parameter do (just for consistency with all the other code)